core/src/main/scala/com/fortysevendeg/mvessel/Connection.scala

```diff
@@ -7,14 +7,13 @@ import java.util.concurrent.Executor
 import com.fortysevendeg.mvessel.logging.{AndroidLogWrapper, LogWrapper}
 import com.fortysevendeg.mvessel.metadata.DatabaseMetaData
 import com.fortysevendeg.mvessel.statement.{PreparedStatement, Statement}
-
-import scala.util.{Failure, Success, Try}
+import com.fortysevendeg.mvessel.util.DatabaseUtils._
 
 class Connection(
```

Comment:
It's possible that you can use `case class` here, and all your parameters will be `val`s 

I don't know how you are using this class. It's only a comment for if you want to try it